src/gmail.js

```diff
@@ -64,7 +64,7 @@ var Gmail = function(localJQuery) {
         console.warn("GmailJS: using deprecated API for old Gmail.", text);
     }
 
-    api.version           = "0.8.0";
+    api.version           = "1.0.17";
```

Comment:
api version != package version. 

Also: this property haven’t really been used (or relevant at all) since support for bundling or stacking was introduced. You can safely omit this change.

Reply:
Will do.